### ISSUE TITLE
Namespace is hardcoded in Cas2TicketValidator

### ISSUE THREAD

**Issue by stefaanvermassen** [IUCrimson/AspNet.Security.CAS#25]
Hi,

Is there a reason the namespace is hardcoded in Cas2TicketValidator? It causes the parsing of the authenticationSuccess message to fail.
```
private readonly XNamespace _ns = "http://www.yale.edu/tp/cas";
```

**Comment by stefaanvermassen** [comment]
Created [PR 26: Add CAS option for namespace and attributes node](https://github.com/IUCrimson/AspNet.Security.CAS/pull/26)

**Comment by davidmdem** [comment]
Not a specific reason. Probably because it's the namespace used in the [response schema documentation](https://apereo.github.io/cas/4.2.x/protocol/CAS-Protocol-Specification.html#4251-saml-cas-response-attributes).

I don't see any problem with it being an option and I'll get the PR merged and update the package in NuGet.

**Comment by davidmdem** [comment]
PR #26 is merged and published to NuGet as version [2.0.5](https://www.nuget.org/packages/AspNetCore.Security.CAS/2.0.5). 

Thanks again and let me know if there are any problems!
